xml/ha_qdevice-qnetd.xml

```diff
@@ -0,0 +1,487 @@
+               Provide at least <option>--qnetd-hostname</option> and the hostname or
+               IP address of the &qnet; server (&node3; in our case):
+            </para>
+            <screen>&prompt.root;<command>crm</command> cluster init qdevice --qnetd-hostname=&node3;</screen>
+            <para>
+               If you want to change the default settings, get a list of all
+               possible options with the command <command
+                  >crm cluster init qdevice --help</command>.
+               All options related to &qdevice; start with
+               <option>--qdevice-<replaceable>NAME</replaceable></option>.
+            </para>
+         </step>
+      </procedure>
+      <para>
+         If you have used the default settings, it will create a &qdevice; with TLS enabled
+         and using the FFSplit algorithm.
+      </para>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-heuristic">
+      <title>Setting up a &qdevice; with Heuristics</title>
+      <para>
+         If you need finer control over how votes are determined, use heuristics.
+         Heuristics are a set of commands which are executed in parallel.
+      </para>
+      <para>
+         For this purpose, the command <command>crm cluster init qevice</command>
+         provides the option <option>--qdevice-heuristics</option>. You can
+         pass one or more commands (separated by semicolon) with absolute paths.
+      </para>
+      <para>
+         For example, if your own command for quorum calculation is located at
+         <filename>/usr/sbin/my-script.sh</filename> you could run it on
+         one of your cluster nodes as follows:
+      </para>
+      <screen>&prompt.root;<command>crm</command
+> cluster init qdevice --qdevice-hostname=&node3; \
+     --qdevice-heuristics=/usr/sbin/my-script.sh</screen>
+      <para>
+         The command can be anything (Shell, Python, Ruby etc). It has to signal the result
+         by returning an exit code. The script returns zero as exit code if the heuristic is
+         correct. If not, the script needs to signal that with an exit code like one (not
+         equal to zero).
+      </para>
+      <para>
+         If you need more than one command, you can do so as well. In this case, only
+         when all commands finish successfully (return code is zero), the heuristics
+         has passed.
+      </para>
+   </sect1>
+
+   <sect1 xml:id="sec-ha-qdevice-status">
+      <title>Checking and Showing Quorum Status</title>
+      <!-- HINT:
+The Ring ID and some other parts are numbers which are actually an IP address.
+You can "decrypt" it by using Python:
+
+$ python3
+>>> import ipaddress
+# Convert a number to an IP address:
+>>> ipaddress.ip_address(3232235778)
+IPv4Address('192.168.1.2')
+# Convert an IP address to a number:
+>>> int(ipaddress.IPv4Address("192.168.1.2"))
+3232235778
+      -->
+      <para>
+         To assess the quorum of your cluster, you can do so on one
```

Comment:
I think many developers don't know the verb assess. I would rather use something like "the get the status" or "to get the information".

Reply:
According to [Merriam Webster](https://www.merriam-webster.com/dictionary/assess) they say:

> 1: to determine the rate or amount of (something, such as a tax, charge, or fine)
> [...]
> 4: to determine the importance, size, or value of

But I agree, it's probably better to use a different word. I will use "get" as you suggested.